include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -648,6 +648,12 @@ class BlmcJointModules
                          
         } while(current_time < (final_time + sampling_period));
 
+        for(unsigned i = 0; i < COUNT; i++)
+        {
+            modules_[i]->set_torque(0.0);
+            modules_[i]->send_torque();
+        }
```

Comment:
why was this block added? i think this should probably be in a separate pull request, and ideally Felix should check that since i am not very clear about how this function works.

Reply:
I figured that the go_to function should send 0.0 torques once arrived at its destination it is not like a permanent control here. We just want to do a visual check if the joint went to the desired pose.